diff -up qtdeclarative-everywhere-src-6.8.1/src/qml/jsruntime/qv4engine.cpp.1~ qtdeclarative-everywhere-src-6.8.1/src/qml/jsruntime/qv4engine.cpp
--- qtdeclarative-everywhere-src-6.8.1/src/qml/jsruntime/qv4engine.cpp.1~	2024-12-05 14:39:43.739771994 +0100
+++ qtdeclarative-everywhere-src-6.8.1/src/qml/jsruntime/qv4engine.cpp	2024-12-05 14:58:48.304236631 +0100
@@ -2208,13 +2208,14 @@ ExecutionEngine::DiskCacheOptions Execut
 {
     if (forceDiskCache())
         return DiskCache::Enabled;
+    return DiskCache::Disabled; /*
     if (disableDiskCache() || debugger())
         return DiskCache::Disabled;
     static const DiskCacheOptions options = qmlGetConfigOption<
             DiskCacheOptions, transFormDiskCache>("QML_DISK_CACHE");
     return hasPreview.loadAcquire()
             ? (options & ~DiskCacheOptions(DiskCache::Aot)) // Disable AOT if preview enabled
-            : options;
+            : options; */
 }
 
 void ExecutionEngine::callInContext(QV4::Function *function, QObject *self,
