diff -up qtdeclarative-everywhere-src-6.7.0-beta1/src/qml/jsruntime/qv4engine.cpp.omv~ qtdeclarative-everywhere-src-6.7.0-beta1/src/qml/jsruntime/qv4engine.cpp
--- qtdeclarative-everywhere-src-6.7.0-beta1/src/qml/jsruntime/qv4engine.cpp.omv~	2024-01-11 14:54:58.101754692 +0100
+++ qtdeclarative-everywhere-src-6.7.0-beta1/src/qml/jsruntime/qv4engine.cpp	2024-01-11 14:55:27.366034565 +0100
@@ -2194,11 +2194,7 @@ ExecutionEngine::DiskCacheOptions Execut
 {
     if (forceDiskCache())
         return DiskCache::Enabled;
-    if (disableDiskCache() || debugger())
-        return DiskCache::Disabled;
-    static const DiskCacheOptions options = qmlGetConfigOption<
-            DiskCacheOptions, transFormDiskCache>("QML_DISK_CACHE");
-    return options;
+    return DiskCache::Disabled;
 }
 
 void ExecutionEngine::callInContext(QV4::Function *function, QObject *self,
